libcove/lib/common.py

```diff
@@ -968,6 +968,8 @@ def get_json_data_missing_ids(json_data_paths, schema_obj):
             for specific_path in json_data_paths[generic_no_id]:
                 if type(specific_path[-1]) != int:
                     continue
+                if type(json_data_paths[generic_no_id][specific_path]) != object:
```

Comment:
I'm not sure `object` is correct here, should it maybe be `dict`? I think as it is everything will be skipped, which suggests we've not got very good tests of this.

Reply:
Oops you're right, I was thinking in JavaScript. Fixed. There are tests for this in cove-ocds, which I have now run and pass.